app/models/ZkKafka.scala

```diff
@@ -40,20 +40,26 @@ object ZkKafka {
   def getTopologies: Seq[Topology] = {
     zkClient.getChildren.forPath(makePath(Seq(stormZkRoot))).asScala.map({ r =>
       getSpoutTopology(r)
-    }).sortWith(topoCompFn)
+    }).flatten.sortWith(topoCompFn)
```

Comment:
I think that modifying `getZkData` to react (log) if it will return a None makes more sense as _where_ that should be, yeah?